lib/core/rollup.config.js

```diff
@@ -0,0 +1,146 @@
+import fs from "fs";
```

Comment:
This config still has some unused parts I think, but it's much better and I think we can merge this after you ensure we can run a simple local dev.

Actually the "rollup" should be for distribution, so it's not really required to be completely correct atm